debian/enterprise-kafka/Dockerfile.rpm

```diff
@@ -30,7 +30,7 @@ EXPOSE 9092
 
 RUN echo "===> installing confluent-support-metrics ..." \
     && yum -q -y update \
-    && yum install -y \
+    && yum install -y --nogpgcheck \
```

Comment:
Stuff like this needs to be automatically toggled from a script via env otherwise it is easy to forget to change